homeassistant/components/netatmo/strings.json

```
@@ -98,6 +100,20 @@
         }
       }
     },
+    "sync_schedule": {
+      "name": "Sync heating schedule",
+      "description": "Syncs the heating schedule for Netatmo climate device.",
+      "fields": {
+        "home_id": {
+          "name": "[%key:component::netatmo::device_automation::trigger_subtype::home_id%]",
+          "description": "Home Id."
```

This is not a description, it's identical to the name.
Add more explanations where to find that ID, for example (change as needed):

```suggestion
          "description": "The ID of your home in the Netatmo mobile app."
```